projects/js-packages/publicize-components/src/components/social-previews/constants.js

```diff
@@ -43,10 +44,10 @@ export const AVAILABLE_SERVICES = [
 		name: 'tumblr',
 		preview: TumblrPreview,
 	},
-	/* {
+	{
 		title: __( 'Mastodon', 'jetpack' ),
 		icon: props => <SocialServiceIcon serviceName="mastodon" { ...props } />,
-		name: 'mastadon',
-		preview: () => null,
-	}, */
+		name: 'mastodon',
+		preview: MastodonPreview,
+	},
```

Comment:
This has been moved to `/components/social-previews/useAvailableServices.js` in #30929 to allow enabling/disabling connections.